fission-web-server/library/Fission/Web/Server/Handler/User/Verify.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
Do we need this here, or was it recommended by the GHC when the code was in an intermediate state?

Reply:
I swear I had GHC barking at me for this once, but it must've been a cascading effect from somewhere else. It's not needed anymore. Removing it :+1: